common/kvstore/KeyValueStore.cc

```diff
@@ -72,7 +72,7 @@ void KeyValueStore::write(string_view key, const vector<u1> &value) {
     int rc;
     rc = mdb_put(txn, dbi, &kv, &dv, 0);
     if (rc != 0) {
-        throw invalid_argument("failed write into database");
+        throw_mdb_error("failed write into database"sv, rc);
```

Comment:
backtrace suggests that that's the one failing, but I made the change to others given that I was here